Framework/Source/API/ConstantBuffer.h

```diff
@@ -133,45 +133,11 @@ namespace Falcor
             return VariablesBuffer::setVariableArray(name, 0, pValue, count);
         }
 
-        /** Set a texture or image.
-            The function will validate that the resource Type matches the declaration in the shader. If there's a mismatch, an error will be logged and the call will be ignored.
-            \param[in] name The variable name in the program. See notes about naming in the ConstantBuffer class description.
-            \param[in] pTexture The resource to bind. If bBindAsImage is set, binds as image.
-            \param[in] pSampler The sampler to use for filtering. If this is nullptr, the default sampler will be used
-        */
```

Comment:
It shouldn't break anything because they never did anything (these methods are leftovers from the OpenGL days). 
In fact, `setTextureInternal()` is an empty function